package.json

```diff
@@ -41,30 +30,44 @@
     "babel-plugin-transform-es2015-unicode-regex": "^6.11.0",
     "babel-plugin-transform-regenerator": "^6.16.1",
     "babel-polyfill": "^6.22.0",
+
+    "babel-plugin-yo-yoify": "^0.3.3",
+    "babel-preset-es2015": "^6.24.0",
+
     "babelify": "^7.3.0",
     "brace": "^0.8.0",
     "browserify": "^13.0.0",
+    "browserify-reload": "^1.0.3",
     "csslint": "^1.0.2",
     "ethereum-remix": "https://github.com/ethereum/remix",
     "ethereumjs-abi": "https://github.com/ethereumjs/ethereumjs-abi",
-    "ethereumjs-block": "^1.2.2",
-    "ethereumjs-tx": "^1.1.1",
-    "ethereumjs-util": "^4.4.0",
-    "ethereumjs-vm": "^2.0.1",
+    "ethereumjs-block": "https://github.com/ethereumjs/ethereumjs-block",
```

Comment:
yes for now, until new releases contain the changes.
The published versions are 2 months or 7 months old.
but cool - merged :-) ... yay